components/src/md3/index.ts

```diff
@@ -1,2 +1,4 @@
 export * from './ChannelValue';
 export * from './ListItemTag';
+export * from './Icon';
```

Comment:
We don't need to expose Icon as its an internal utility for our components, but it's fine for now